lib/postgrex/notifications.ex

```diff
@@ -146,7 +146,7 @@ defmodule Postgrex.Notifications do
 
     * `:timeout` - Call timeout (default: `#{@timeout}`)
   """
-  @spec listen(GenServer.server(), String.t(), Keyword.t()) ::
+  @spec listen(:gen_statem.server_ref(), String.t(), Keyword.t()) ::
```

Comment:
`@type server :: :gen_statem.server_ref()` at the top?

Reply:
Sure, done ✅